tests/src/test/java/alluxio/hadoop/FileSystemAclIntegrationTest.java

```diff
@@ -396,14 +404,12 @@ public void parentDirectoryPermissionForUfsTest() throws IOException {
   }
 
   /**
-   * Tests for the permission of object storage UFS files.
+   * Tests for the permission of S3 UFS files.
```

Comment:
These javadocs don't say anything that the test name doesn't, so they could be removed